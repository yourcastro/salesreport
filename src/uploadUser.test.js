import React from 'react';
import renderer from 'react-test-renderer';
 
import UploadUser from './uploadUser';
 
describe('Sales Report Upload', () => {
  test('snapshot renders', () => {
    const component = renderer.create(<UploadUser />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});