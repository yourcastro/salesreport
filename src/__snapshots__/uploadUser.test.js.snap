// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Counter snapshot renders 1`] = `
<div>
  <div
    className="fileUploadContainer"
  >
    <div>
      <input
        onChange={[Function]}
        type="file"
      />
      <button
        onClick={[Function]}
      >
        Upload
      </button>
    </div>
    <div
      className="uploadMobile"
    >
      <br />
      <h4>
        Please select file before pressing the upload button
      </h4>
    </div>
  </div>
  <div />
  <div
    className="Toastify"
  />
</div>
`;

exports[`Sales Report Upload snapshot renders 1`] = `
<div>
  <div
    className="fileUploadContainer"
  >
    <div>
      <input
        onChange={[Function]}
        type="file"
      />
      <button
        onClick={[Function]}
      >
        Upload
      </button>
    </div>
    <div
      className="uploadMobile"
    >
      <br />
      <h4>
        Please select file before pressing the upload button
      </h4>
    </div>
  </div>
  <div />
  <div
    className="Toastify"
  />
</div>
`;
